addons/survey/views/survey_templates.xml

```diff
@@ -327,7 +325,7 @@
         <t t-set="default_constr_error_msg">This question requires an answer.</t>
         <t t-set="default_validation_error_msg">The answer you entered is not valid.</t>
         <t t-set="default_comments_message">If other, please specify:</t>
-        <div t-att-class="'js_question-wrapper pb-4 %s %s' % ('d-none' if not display_question else '', 'me-2' if extra_right_margin else '')"
+        <div t-att-class="'js_question-wrapper pb-4 %s %s %s' % ('d-none' if not display_question else '', 'me-2' if extra_right_margin else '', 'o_survey_center_align' if survey.questions_layout != 'page_per_question' else '')"
              t-att-id="question.id" t-att-data-required="question.constr_mandatory"
              t-att-data-constr-error-msg="question.constr_error_msg or default_constr_error_msg"
```

Comment:
I see that this will be set even if the question is not mandatory, could we clean the template to avoid this unnecessary content while we are here? Maybe this works?
```suggestion
             t-att-data-constr-error-msg="question.constr_error_msg or default_constr_error_msg if question.constr_mandatory else None"
```